SPECS/mock/mock.spec

```diff
@@ -152,7 +153,7 @@ Requires(pre): shadow-utils
 Filesystem layout and group for Mock.
 
 %prep
-%setup -q -n mock-%{name}-%{version}-1/%{name}
+%autosetup -p2 -n mock-%{name}-%{version}-1/%{name}
```

Comment:
Can we standardize on -p1 which is more typical?  Just thinking about potential future patch work.

Reply:
For the sake of posterity, the short version is that the upstream `mock` repo has 2 subdirs (`mock` and `mock-core-configs`). Each carries its own `.spec` and is intended to be packaged separately. Unfortunately, patches directly downloaded from github are relative to the root of the repo, and end up 1 extra dir step away from the sources that we download--hence the `-p2`.